projects/plugins/beta/class-jetpack-beta-admin.php

```diff
@@ -168,12 +168,30 @@ public static function to_test_content() {
 			case 'pr':
 				return self::to_test_pr_content( $branch );
 			case 'master':
+				return self::to_test_general_rules_content();
 			case 'rc':
 				return self::to_test_file_content();
 		}
 		return null;
 	}
 
+	/**
+	 * General rules and recommendations for new Beta Testers.
+	 * Displayed when no specific branch is picked.
+	 *
+	 * @since 2.5.0
+	 */
+	public static function to_test_general_rules_content() {
+		$test_rules = WP_PLUGIN_DIR . '/' . Jetpack_Beta::get_plugin_slug() . '/docs/testing/testing-tips.md';
+		if ( ! file_exists( $test_rules ) ) {
+			return;
+		}
+		WP_Filesystem();
+		global $wp_filesystem;
+		$content = $wp_filesystem->get_contents( $test_rules );
+		return self::render_markdown( $content );
+	}
+
 	/** Return testing instructions for release candidate branch */
 	public static function to_test_file_content() {
 		$test_file = WP_PLUGIN_DIR . '/' . Jetpack_Beta::get_plugin_slug() . '/to-test.md';
```

Comment:
For some reason, this $test_file keeps returning as false for me, so nothing renders. Not sure if something is funky with docker and testing that I'm missing?